src/renderer/main/package.js

```diff
@@ -792,9 +792,18 @@ async function uninstallPackage(instPath) {
     if (!file.isInstallOnly)
       filesToRemove.push(path.join(instPath, file.filename));
   }
-  await Promise.all(
-    filesToRemove.map((filePath) => safeRemove(filePath, instPath))
-  );
+
+  try {
+    await Promise.all(
+      filesToRemove.map((filePath) => safeRemove(filePath, instPath))
+    );
+  } catch {
+    buttonTransition.message(btn, 'エラーが発生しました。', 'danger');
```

Comment:
```suggestion
  } catch (e) {
    log.error(e);
    buttonTransition.message(btn, 'エラーが発生しました。', 'danger');
```
Please log all caught errors.

Reply:
Fixed.